src/iuse.cpp

```diff
@@ -4683,11 +4683,6 @@ int iuse::set_trap(player *p, item *it, bool, point)
         return 0;
     }
 
-    if (dirx == p->posx() && diry == p->posy()) {
```

Comment:
That'll do.  Thanks: I'm coming off a busy week of traveling and business and would really rather be taking rec time about now.  But these ain't gonna mergetest themselves.  ;-)